### ISSUE TITLE
[Screen Reader - Power Apps Admin- Analytics]: In scan mode, user is not able to navigate further over the page contents using down arrow key.

### ISSUE THREAD

**Issue by rishuSrivastava11hcl** [highcharts/highcharts#14513]
### Test Environment:

- Version: 2004 (OS Build: 19041.508).
- Browser: Edge chromium Version 87.0.634.0 (Official build) dev (64-bit).
- Screen Reader: Narrator
- #URL:https://admin.powerplatform.microsoft.com/

### **User Experience:**
Screen reader user would not be able to navigate any page if the Narrator focus is not in sync with keyboard focus, due to this they will miss the functionality of controls appears on the page.

### **Repro Steps:​**

1. Open URL: https://admin.powerplatform.microsoft.com/ and login with valid credentials.
2. Power Platform Admin center page will get appear, navigate to 'Analytics' tab and expand it, Select Common Data Services tab.
3. Enable the narrator and navigate to all controls present under 'Active Users' tab in scan mode using down arrow key.
4. Observe if the user is able to navigate further after the chart present for 'Active users performing specific operations' or not.
5. Verify the issue.

### **Actual Result:**
In scan mode, user is not able to navigate further after the chart present for 'Active users performing specific operations' over the page contents using down arrow key.

### **Expected Result:**
In scan mode, user should be able to navigate further over all the page contents using down arrow key to read and access.

### **Note:**

1. Same issue does not exist with 'Firefox + NVDA' browse mode.
2. This issue exists throughout the application under similar tab controls.

### **WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Screenshots N/A**

**Comment by rishuSrivastava11hcl** [comment]
#Accessibility;#WCAG4.1.2;#HCL;#HCL-PowerAppsAdminCenter;#A11ySev2;#Benchmark

**Comment by oysteinmoseng** [comment]
Hi @rishuSrivastava11hcl, are you able to provide a [jsFiddle](https://jsfiddle.net/) demo reproducing the issue for us to debug? I am not able to log in to the portal linked in the issue.

**Comment by rishuSrivastava11hcl** [comment]
yes @oysteinmoseng

**Comment by sebastianbochan** [comment]
Closed due to inactivity.

**Comment by rishuSrivastava11hcl** [comment]
> Hi @rishuSrivastava11hcl, are you able to provide a [jsFiddle](https://jsfiddle.net/) demo reproducing the issue for us to debug? I am not able to log in to the portal linked in the issue.

![image](https://user-images.githubusercontent.com/72869551/117443012-299a6f80-af55-11eb-9853-4427df81b3a4.png)
 please observe the issue in above image

**Comment by oysteinmoseng** [comment]
@rishuSrivastava11hcl This seems to be an issue with how Narrator deals with SVG accessibility, and unfortunately not something that we can control on our end - unless the Narrator team has specific guidelines they want us to follow. You can get around the issues by using `tab`, `Ctrl+Down arrow` and/or `Ctrl+Left arrow` to move beyond some of the areas where Narrator gets stuck, but this does not appear to be an issue with other screen readers. It also behaves differently with Narrator on Chrome.

I am closing for now, as there does not seem to be much we can do on our end here, but feel free to comment to reopen if you have more info for us.
